src/mono/msbuild/apple/build/AppleBuild.props

```diff
@@ -2,7 +2,7 @@
   <!-- iOS/tvOS device + arm64 simulators need to AOT -->
   <PropertyGroup Condition="'$(TargetOS)' == 'ios' or '$(TargetOS)' == 'tvos' or (('$(TargetOS)' == 'iossimulator' or '$(TargetOS)' == 'tvossimulator') And '$(TargetArchitecture)' == 'arm64')">
     <RunAOTCompilation Condition="'$(RunAOTCompilation)' == ''">true</RunAOTCompilation>
-    <iOSLikeDedup>true</iOSLikeDedup>
+    <iOSLikeDedup Condition="'$(iOSLikeDedup)' == ''">true</iOSLikeDedup>
```

Comment:
Looks like this file will be part of library builder, there is still a bunch of references to App in this file, since its more generic and shared, maybe we should make sure properties are aligned as much as possible to be more generic and not app centric?

Reply:
Modified to be more generic.